<launch>
	<node pkg="vicon_demo" type="tracking_demo" name="tracker" output="screen">
                <param name="vicon_array_topic" value="/viconArray" type="str" />
                <param name="advertise_topic" value="tracker_demo" type="str" />
                <param name="transform_topic" value="vicon/swarmbot_AA/swarmbot_AA" type="str" />
                <param name="lemniscate_constant" value="50" type="int" />
                <param name="lemniscate_interval" value="10" type="int" />
                <param name="cutoff_radius" value="5" type="int" />
		
		</node>
</launch>
